test/utils_spec.js

```diff
@@ -1257,6 +1257,10 @@ describe("utils", function () {
   it("should not add version if public_id contains version already", function () {
     test_cloudinary_url("v1234/test", {}, `http://res.cloudinary.com/${cloud_name}/image/upload/v1234/test`, {});
   });
+  it("should not add version if exclude_version:true", function () {
```

Comment:
@strausr , there are more tests that can be done here. 
For example - exclude_version = false or exclude_version=true and version !=null. 
Please check in other SDKs